spark/src/main/java/com/nflabs/zeppelin/spark/SparkSqlInterpreter.java

```diff
@@ -76,13 +76,20 @@ public void open() {
     this.maxResult = conf.getInt("ZEPPELIN_SPARK_MAX_RESULT",
         "zeppelin.spark.maxResult",
         Integer.parseInt(getProperty("zeppelin.spark.maxResult")));
+    openSparkInterpreter(true);
```

Comment:
@guptarajat  
Thanks for explanation. 
Still I think we can have some better codes. 
What if we just open it everytime, if it's safe and not triggering some heavy operation? 
It makes our code simpler.